modules/hp_graph/README.md

```diff
@@ -0,0 +1,18 @@
+# Howard Paragraphs Graph
+
+Provides a paragraphs bundle for adding Bar Graphs as a Kitchen Sink Widget.
+This module contains markup only (no js or css), those should be provided in the client theme, loaded via the idfive Component Library:
+
+- [idfive Component Library](https://bitbucket.org/idfivellc/idfive-component-library)
+- [idfive Component Library D8 Theme](https://bitbucket.org/idfivellc/idfive-component-library-d8-theme)
+
+## Configuration
+
+The following fields are available on this KS widget:
+
+- XX: Description.
```

Comment:
@ben2collins please fill this out for all fields. It will help admins/etc understand what this module does/etc. and how fields affect what is going on.

Reply:
I filled out all of the fields and pushed the changes  to my latest commit.